src/components/tables/TitleWithTags.tsx

```diff
@@ -33,7 +35,10 @@ const TitleWithTags = ({
   return (
     <Td width="40%">
       <Flex gap={2} flexDir="column">
-        <Text>{title}</Text>
+        <div>
```

Comment:
lets use chakra `Box` to follow the repo coding style
```suggestion
        <Box>
```